cmd/di.go

```diff
@@ -465,11 +466,39 @@ func (di *Dependencies) bootstrapIdentityComponents(options node.Options) {
 	di.IdentityRegistration = identity_registry.NewRegistrationDataProvider(di.Keystore)
 }
 
-func (di *Dependencies) bootstrapLocationComponents(options node.OptionsLocation, configDirectory string) (err error) {
+func (di *Dependencies) bootstrapLocationComponents(options node.OptionsLocation, configDirectory string) error {
```

Comment:
@vkuznecovas extracted location di to the relevant package, I don't think it's a good idea to bring this back

Reply:
I agree, at least some explanation for reasons required.

Reply:
It's not good idea to make it in `location` package, because it's independent package, and making it modular leads us to circular dependencies like `cmd -> node -> location -> node`

I don't see a problem with `di` being high-level entrypoint to glue dependencies. Anyway moved to just 1 factory function  to `location/factory`.

Reply:
The issue is that other packages are also not yet well designed, but we got to start somewhere. I wouldn't try to find reasons why current `di` is ok, because it's awful order dependent spaghetti. We should split it feature-wise, and perhaps separate object graph construction from bootstrap, so that construction is not order-dependent, but bootstrap _is_.
I think putting it in a subpackage _for now_ is ok. However, _factory_, aliasing adds just another layer of obfuscation.

Reply:
Modularisation is not scope of this PR